templates/web/thesaurus/thesaurus.html.twig

```diff
@@ -315,6 +323,29 @@
 
         <script type="text/javascript">
 
+            //Ajax function to get data
```

Comment:
cool but check there is no multi-modal case in app, e.g. a (level 2) confirm dialog opened by a (level 1) edit dialog (to be checked...)
If there is such a case (?), solution : pass "#NEWSY_DLG..." as an arg to function.